jira/client.py

```diff
@@ -3533,7 +3533,7 @@ def search_issues(
               If maxResults evaluates to False, it will try to get all issues in batches. (Default: ``50``)
             validate_query (bool): True to validate the query. (Default: ``True``)
             fields (Optional[Union[str, List[str]]]): comma-separated string or list of issue fields to include in the results.
-              Default is to include all fields.
+              Default is to include all fields. If you don't require fields, set it to empty string ``''``.
```

Comment:
search issues has been deprecated and slightly reworked
i'd suggest implementing it at enhanced_search_issues.